src/utils/initialization.js

```diff
@@ -47,9 +47,9 @@ Now, let's get started!\n`)
     answer = await promptConfirm(`Would you like to create the code and content under your github username, "${org}"?`)
   }
   if (!answer) {
-    org = await promptInput('Enter the organization under which to create the code and content:')
+    org = await promptInput('Enter the organization name (e.g my-company) under which to create the code and content:')
```

Comment:
Enter the GitHub organization name (e.g., my-company) to create the repository under: